tests/code/test_users.py

```diff
@@ -103,9 +104,37 @@ def test_removeUserInterestt():
         transaction.rollback()
 
 @pytest.mark.integration
-def test_banUser():
+def test_addUserProfileNote():
+    with mainDB.atomic() as transaction:
+        with app.app_context():
+            g.current_user = "ramsayb2"
+            profileNote = addProfileNote(1, True, "Test profile note", "neillz")
+            assert profileNote == ProfileNote.get_by_id(profileNote.id)
+            profileNote2 = addProfileNote(3, False, "Test profile note 2", "ramsayb2")
+            assert profileNote2 == ProfileNote.get_by_id(profileNote2.id)
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_deleteUserProfileNote():
     with mainDB.atomic() as transaction:
+        with app.app_context():
+            g.current_user = "ramsayb2"
+
+            addedNote = addProfileNote(1, True, "Test profile note", "neillz")
+            profileNote = deleteProfileNote(addedNote)
```

Comment:
We should test once between these to make sure that the note actually exists in the db with the content we expect.